src/ximea_driver.cpp

```diff
@@ -6,20 +6,50 @@ namespace trailbot {
 
 constexpr int kTimeoutMs = 100;
 
-// Fixed exposure time
-// TODO(jacob): Make these parameters
-constexpr int kExposureMs = 50;
-constexpr int kExposureUs = kExposureMs * 1000;
-
 XIMEADriver::XIMEADriver() : Node("ximea_driver") {
-  pub_ = create_publisher<sensor_msgs::msg::Image>("camera", 10);
```

Comment:
Is there a reason why publisher is not created in the constructor?

Reply:
The publisher is created from the `ImageTransport`, which requires the node to be properly initialized, which only happens after the constructor is complete.